pkg/sectigo/sectigo.go

```diff
@@ -124,6 +125,14 @@ func New(conf Config) (client *Sectigo, err error) {
 		if conf.Password == "" {
 			conf.Password = MockPassword
 		}
+
+		if conf.Endpoint != "" {
+			var u *url.URL
+			if u, err = url.Parse(conf.Endpoint); err != nil {
+				return nil, fmt.Errorf("could not parse sectigo endpoint: %w", err)
+			}
+			SetBaseURL(u)
+		}
```

Comment:
The idea here is that if you're in testing mode you can specify an endpoint - but we'll have to do something about the `localhost` guard that we have so that we can deploy this in staging. Perhaps setting a staging flag instead of a testing flag?

Reply:
That could work.